sonar-html-plugin/src/main/java/org/sonar/plugins/html/checks/coding/UnclosedTagCheck.java

```diff
@@ -43,8 +43,18 @@
   private List<String> ignoreTagsList;
   private final List<TagNode> nodes = new ArrayList<>();
 
+  private boolean skipFile = false;
+
   @Override
   public void startDocument(List<Node> nodes) {
+    skipFile = false;
+    if (getHtmlSourceCode().inputFile().filename().endsWith(".cshtml")) {
```

Comment:
You can extract this line into separate method with meaningful name. Will make code more readable.